### DISCUSSION TITLE
How would you test ?

### DISCUSSION THREAD

**Discussion by gaetan-wap** [BitPhinix/slate-yjs#249] [Q&A] [answered]
First, thanks for making this project.

I have been using slate-yjs and [udecode/slate-plugins](https://github.com/udecode/slate-plugins) with custom code. 

I am experiencing weird behaviours from time to time.

I have a hard time reproducing these bugs locally and deterministically (as someone would expect with a distributed system).

How would you test such an editor ?

**Comment by BitPhinix** [comment, marked as answer]
Hi @gaetan-wap,

Testing collaborative editors is **really hard**. The best approach would probably be spinning up multiple collaborative editor instances and fuzzing them using a self-build fuzzer.

**Reply by gaetan-wap** [reply]
That's what I feared. Thanks.
